gubernator/github/classifier.py

```diff
@@ -115,16 +115,14 @@ def get_labels(events, labels=None):
                     logging.warning('label event with no labels (multiple changes?)')
                 elif action == 'labeled':
                     label = body['label']
-                    if label not in labels:
-                        labels.append(label)
+                    if label['name'] not in labels:
+                       labels[label['name']] = label['color']
                 elif action == 'unlabeled':
-                    label = body['label']
-                    if label in labels:
-                        labels.remove(label)
+                    labels.pop(body['label']['name'], None)
```

Comment:
should this be `del labels[body['name']]` ?

Reply:
This should work. `pop` is defined for dictionaries in python. Specifying `None` as a default is unnecessary though.

Reply:
Using `del` or omitting `None` causes a KeyError-- the point is to remove a key whether or not it exists.